### DISCUSSION TITLE
How do you make a button with a gradient in Bootstrap 5?

### DISCUSSION THREAD

**Discussion by nreef12** [twbs/bootstrap#37831] [Q&A] [answered]
I'm kinda new to Bootstrap, so I'm wondering: how do you make a button with a gradient in Bootstrap 5, like in Bootstrap 4?

**Comment by tbolt** [comment, marked as answer]
Should be the same: https://getbootstrap.com/docs/5.3/utilities/background/#background-gradient

You could always do a custom gradient with [linear-gradient](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient).

**Reply by nreef12** [reply]
Thx
